### PR TITLE
Move all maintainers scripts to use symfony/console - `zf-maintainer`

### PR DESCRIPTION
**Pull request by webimpress**
- [x] changelog-bump
- [x] lts-release
- [x] lts-components
- [x] lts-patch
- [x] lts-stage
- [x] sync-repos

### CODE REVIEW

**Comment by webimpress** [comment]
@weierophinney It's done now !

File: LTS-WORKFLOW.md [hunk 1]
```diff
@@ -162,5 +162,5 @@ Now that the tag is made, the Zend team will need to build and release distribut
 such, please coordinate with them whenever you tag, so that they can do so as soon as possible after
 a tag is created.
 
-If you cannot find one of them in the usual IRC channels, please send an email to
-zf-deveam@zend.com.
+If you cannot find one of them in the [Slack](https://zendframework-slack.herokuapp.com),
+please send an email to zf-deveam@zend.com.
```
Comment (weierophinney) [inline LTS-WORKFLOW.md]: Note to self: `s/deveam/devteam/` here.

File: LTS-WORKFLOW.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ $ ../vendor/bin/phpunit ZendTest/View/Helper/ServerUrl.php
 > If you observe failing tests when there shouldn't be, run `composer update` and then re-run the
 > tests.
 
-At this point, you can now tag; the `lts-stage` will provide you with the `git tag` command to
+At this point, you can now tag; the `lts:stage` will provide you with the `git tag` command to
```
Comment (weierophinney) [inline LTS-WORKFLOW.md]: Note to self: add "target" following `lts:stage`.

File: README.md [hunk 3]
```diff
@@ -54,12 +54,23 @@ repository. Run the following:
...
 
-### `bin/zf-maintainer`
-
-This script currently has the following actions:
+This script currently has the following general use command:
+- `changelog-bump <version> [-b|--base <master|develop>]` - bumps the
+  CHANGELOG version for the current component;
 - `create-package <name>` - creates a new package from templates;
-- `rebase-doc-template <path>` - rebases all templates for specified package.
+- `rebase-doc-template [<path>]` - rebases all templates in package,
```
Comment (weierophinney) [inline README.md]: Note to self: `s/,/;/`

File: README.md [hunk 4]
```diff
@@ -54,12 +54,23 @@ repository. Run the following:
...
+  by default current directory is used;
```
Comment (weierophinney) [inline README.md]: Note to self: add a `,` following `by default`.

File: README.md [hunk 5]
```diff
@@ -54,12 +54,23 @@ repository. Run the following:
...
+- `sync-repos <github-token>` - synchronizes zendframework packages descriptions;
+
+and command to help maintain LTS release:
+- `lts:components` - lists LTS components, one per line;
+- `lts:patch -p|--patchfile=<patchfile> -t|--target=<target> -c|--component=<component>` -
+  rewrites a component patch so it may be applied against the ZF2 repo;
+- `lts:release <version> -e|--exclude=<exclude> [-b|--basePath <base-path>]` -
+  tags a new LTS maintenance release of all components;
```
Comment (weierophinney) [inline README.md]: Note to self: end each bullet point with a `.` instead of a `;`.

**Pull request merged**